### DISCUSSION TITLE
希望提供支持 简书的 https://www.jianshu.com/techareas/frontend

### DISCUSSION THREAD

**Discussion by radiorz** [DIYgod/RSSHub#10088] [Q&A] [answered]
https://www.jianshu.com/techareas/frontend

**Comment by TonyRL** [comment, marked as answer]
> 请使用issue进行反馈

_Originally posted by `@NeverBehave` in https://github.com/DIYgod/RSSHub/discussions/9943#discussioncomment-2949818_
